string bin_type        # Type of the bin
string[] bin_contents  # Contents of the bin
string target_object   # Object to be removed
float64[] start_joints # Initial joint angles
string strategy        # Strategy used for removing object from bin